templates/net/configs/netplan.yaml.j2

```diff
@@ -1,27 +1,51 @@
-
-    # {{ name }} Interface
-    {{ identifier }}:
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    {{ system_name }}:
       dhcp4: false
       dhcp6: false
       accept-ra: no
-{%- if ips != None and ips|length > 0 %}
-\n      addresses:
-{%- for ip in ips %}
-\n        - {{ ip }}
-{%- endfor %}
-{%- endif %}
-{%- if routes != None and routes|length > 0 %}
-\n      routes:
-{%- for route in routes %}
-\n        - to: {{ route['to'] }}
-          via: {{ route['via'] }}
-{%- endfor %}
-{%- endif %}
-{%- if mac != None %}
-\n      match:
+      set-name: {{ standard_name }}
+      {%- if mac is not none %}
```

Comment:
I believe the newline `\n` is still required inside if and for loops. I tested locally this template with dummy data and here was the output. 

```
network:
  version: 2
  renderer: networkd
  ethernets:
    eth0:
      dhcp4: false
      dhcp6: false
      accept-ra: no
      set-name: public0      match:
        macaddress: a2:00:34:23:45:32      addresses:        - 10.0.0.2/24        - 10.0.0.3/24      routes:        - to: 10.0.0.1/24
          via: 10.0.0.1  vlans:    public0.404:
      dhcp4: false
      dhcp6: false
      accept-ra: no
      id: 404
      link: public0      addresses:        - 10.10.10.2        - 10.10.10.3      routes:        - to: 10.10.10.1/24
          via: 10.10.10.1
```

Reply:
okay, I was correct before but gave a try with chatGPT for clean look but of no use, will put new lines wherever required.

Reply:
changed, can you test it now please?

Reply:
Issue is not with the newlines, can you remove them. 

The problem is with the `if` and `for` statements. 

1. Do not indent 
2. Start the, with `{% ` instead of `{%- `

Reply:
done